### PR TITLE
[fortinet_fortigate]Adapt event.module value

### LINKED ISSUE #8453: Fortinet Fortigate Integration Event Module Name
Right now, the event module value for the Fortinet Fortigate integration is `fortinet`.

https://github.com/elastic/integrations/blob/ce2f88171a247b9bbeedc9035ec5204f91b35daa/packages/fortinet_fortigate/data_stream/log/fields/base-fields.yml#L13

Should the event module name for the integration be `fortinet_fortigate`, similar to the Cisco ASA integration?

https://github.com/elastic/integrations/blob/ce2f88171a247b9bbeedc9035ec5204f91b35daa/packages/cisco_asa/data_stream/log/fields/base-fields.yml#L13

### LINKED ISSUE #8843: FIM Integration: Event.Dataset value not ECS
Right now, the `event.module` value for the FIM integration is `file_integrity`.

https://github.com/elastic/integrations/blob/359eeb6f298ce472692effad35b0ee2f657169e4/packages/fim/data_stream/event/fields/base-fields.yml#L13

And the `event.dataset` is set to `fim.event`

https://github.com/elastic/integrations/blob/359eeb6f298ce472692effad35b0ee2f657169e4/packages/fim/data_stream/event/fields/base-fields.yml#L17

I believe the `event.dataset` should actually be set to: `file_integrity.event`

From ECS docs:

`It’s recommended but not required to start the dataset name with the module name, followed by a dot, then the dataset name.`

This is how all other integrations are setup.

### PR DESCRIPTION
**Pull request by bhapas**
## Proposed commit message

See Title

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #8453

### CODE REVIEW

File: packages/fim/data_stream/event/fields/base-fields.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 - name: event.dataset
   type: constant_keyword
   description: Event dataset
-  value: fim.event
+  value: file_integrity.event
```
[the remark this answers is not in the document]
Comment (P1llus) [reply packages/fim/data_stream/event/fields/base-fields.yml:17]: fim.event is the correct value, its event.module that needs to be changed to "fim".

The module is the package name, while dataset is the combination of package name + datastream name, so if we wanted this to be file_integrity then the package should have been named that as well.

If there is any objection to it then I would check with @andrewkroh, I know the input itself is named file_integrity, but the package is named fim.

**Review by P1llus — COMMENTED** [review]
I added some comments, feel free to approach it the way you feel is best.

I do not know how constant_keywords will react on this however, when a datastream is rolled over I assume its fine, but since that field only accept 1 unique value, then maybe it will break the aggregations between old and new data?

File: packages/fim/data_stream/event/fields/base-fields.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bhapas) [reply packages/fim/data_stream/event/fields/base-fields.yml:17]: Removing `fim` from this PR as it get's the `event.module` from beat input.

**Review by kcreddy — APPROVED** [review]
LGTM 👍🏼 

It would be nice to verify Marius' comment.
> I do not know how constant_keywords will react on this however, when a datastream is rolled over I assume its fine, but since that field only accept 1 unique value, then maybe it will break the aggregations between old and new data?

**Comment by nicpenning** [comment]
FYI - The PR referencing this change is incorrect - Likely from Copy/Paste from previous PR.

Should be https://github.com/elastic/integrations/pull/8853 instead of 8670
image

**Pull request merged**